include/open62541/server_pubsub.h

```diff
@@ -225,6 +225,12 @@ struct UA_PubSubConfiguration {
     void (*stateChangeCallback)(UA_Server *server, UA_NodeId *id,
                                 UA_PubSubState state, UA_StatusCode status);
 
+    UA_Boolean isDeltaFramesEnabled;
+
+#ifdef UA_ENABLE_PUBSUB_INFORMATIONMODEL
+    UA_Boolean isInformationModelMethodsEnabled;
```

Comment:
We don't habe the `isSomething` idiom anywhere else.
Instead:

`enableDeltaFrames`
`enableInformationModel`
`enableInformationModelMethods`